apache/superset
# test(storybook): fix component stories

test(storybook): Fix component story controls
### SUMMARY

Storybook controls for some components were not working. This pr aims to fix that.
### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS

1. Run storybook
2. Check component controls that are previously uninteractable
### ADDITIONAL INFORMATION


- [x] Has associated issue:    #29244
